queries.sql

```diff
@@ -84,3 +84,59 @@ SELECT species,
 FROM animals
 WHERE date_of_birth BETWEEN 'Jan 01, 1990' AND 'Dec 31, 2000'
 GROUP BY species;
+
+
```

Comment:
- [ ] Your readme file looks like a template :smile: because it doesn't have any information regarding you and your application :thinking: Kindly insert your application information in the respective sections to make your readme file descriptive and professional :blush:

Hint: You can see this [readme](https://github.com/samlorlah/recipe-app/blob/dev/README.md) file to get an idea :+1: